packages/cluster-ui/src/statementDetails/diagnostics/diagnosticsView.tsx

```diff
@@ -130,16 +158,11 @@ export class DiagnosticsView extends React.Component<
                       record.statement_fingerprint,
                     )
                   }
+                  className={cx("download-bundle-button")}
                 >
-                  <Button
-                    size="small"
-                    type="flat"
-                    iconPosition="left"
-                    icon={<Download />}
-                  >
-                    Bundle (.zip)
-                  </Button>
-                </a>
+                  <Download />
```

Comment:
You could also use the `Icon` component from `ui-components`.

```jsx
import { Icon } from "@cockroachlabs/ui-compoents";

<Icon iconName="Download" />
```

This is only sugar on top to conform to standard sizes or hues which doesn't seem necessary here.